packages/table/src/headers/rowHeaderCell.tsx

```diff
@@ -56,23 +59,32 @@ export class RowHeaderCell extends React.Component<IRowHeaderCellProps, IRowHead
     };
 
     public render() {
-        const { className, isActive, isRowSelected, name, resizeHandle, style } = this.props;
-        const classes = classNames(className, "bp-table-header", {
+        const { className, isActive, isLoading, isRowSelected, name, resizeHandle, style } = this.props;
+        const classes = classNames(className, "bp-table-header", "bp-table-row-header", {
             "bp-table-header-active": isActive || this.state.isActive,
             "bp-table-header-selected": isRowSelected,
+            "bp-table-row-header-loading": isLoading,
         });
 
```

Comment:
I see how this is working, but i think we should try to find a way to avoid the duplication of rendering on every component that has isLoading.